data/coalition missions.txt

```diff
@@ -18,6 +18,7 @@ mission "Coalition: First Contact"
 	on offer
 		"reputation: Coalition" += 10
 		"reputation: Heliarch" += 10
+		log "Made contact with the Coalition, an alliance of giant beetles, giant spiders and centaurs in the galactic south. They're ruled by 'Heliarchs', and they overthrew the Quarg. Seems they see them as oppressors."
```

Comment:
Saying that they are `ruled by 'Heliarchs'` makes the term "Heliarch" feel too generic. It should say something like `ruled by 'the Heliarchs'`, with a definite article.